tests/regression/irs/first.json

```diff
@@ -3997,6 +3997,32 @@
                             },
                             "interface": null
                         },
+                        {
+                            "type": "Name",
+                            "name": "datum.is_a_good_thing",
+                            "basename": "datum",
+                            "location": {
+                                "lineno": 2,
+                                "col_offset": 7,
+                                "end_lineno": 2,
+                                "end_col_offset": 28,
+                                "file": "/home/suade/rattr/feature/tests/regression/code/module/foo.py"
```

Comment:
Yeah, it is accounted for in the test here:

![image](https://github.com/SuadeLabs/rattr/assets/9117434/4a384610-7619-418f-8a8b-c21dc5af5205)

Bit tricky to avoid as the symbols are immutable :(

I could try to mock `get_current_file` to return a relative file but this doesn't seem to work.